bundles/subclipse.core/src/org/tigris/subversion/subclipse/core/resourcesListeners/FileModificationManager.java

```diff
@@ -315,15 +315,15 @@ public void propertyChange(PropertyChangeEvent event) {
 			ignoreManagedDerivedResources = SVNProviderPlugin.getPlugin().getPluginPreferences().getBoolean(ISVNCoreConstants.PREF_IGNORE_MANAGED_DERIVED_RESOURCES);
 		}
 	}
-	
+
 	private class RefreshStatusCacheSchedulingRule implements ISchedulingRule {
```

Comment:
If we _do_ go back to using `MultiRule` then we should remove the private `RefreshStatusCacheSchedulingRule` class, as it will be no longer used.